homeassistant/components/sensor/__init__.py

```diff
@@ -38,7 +40,9 @@
     DEVICE_CLASS_BATTERY,  # % of battery that is left
     DEVICE_CLASS_CURRENT,  # current (A)
     DEVICE_CLASS_ENERGY,  # energy (kWh, Wh)
+    DEVICE_CLASS_FORCE,  # force (N)
```

Comment:
Please open an issue in our architecture repo for discussion and approval.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Thanks, filed https://github.com/home-assistant/architecture/issues/491 with more context on why I added both `force` and `mass`.

Reply:
Hi @MartinHjelmare, is there any process for getting the issue above reviewed?